business/workloads.go

```diff
@@ -599,26 +587,78 @@ func parseZtunnelLine(line string) *LogEntry {
 		Severity:      "INFO",
 	}
 
+	regex := regexp.MustCompile(models.ZtunnelRegexLine)
```

Comment:
You should pre-compile regexes by just having them be globals like this: https://github.com/kiali/kiali/blob/6bbef003ec097d104a9120588841eada9937fd75/status/versions.go#L28-L32.